setup.cfg

```diff
@@ -36,3 +36,13 @@ multi_line_output = 3
 known_first_party = robotd
 default_section = THIRDPARTY
 sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+
+
+[mypy]
+warn_unused_ignores = True
+strict_optional = True
+
+disallow_any_generics = True
+check_untyped_defs = True
+
+ignore_missing_imports = True
```

Comment:
Is this needed? I'd greatly enjoy inverting this if possible.

Reply:
Indeed, it's rather unfortunate that we need to ignore some imports there (notably I think it was cffi, numpy and PIL at least which don't have typeshed entries), however this project doesn't have any such dependencies, so I'd hope that we could enforce typing everywhere.

Reply:
This is resolved in #59 by 5a08e1a.